pkg/activator/README.md

```diff
@@ -1,52 +1,10 @@
-# The Activator Network Configurations
+# About Activator
 
-## Traffic configuration overview
+The name *activator* is actually a misnomer, since after Knative 0.2,
+*activator* no longer activates inactive Revisions.  The only functionalities of
```

Comment:
Also a couple newlines before "The only functionalities..." would look better rendered, I think.